templates/examples/audio/ctc_asr.md

```diff
@@ -0,0 +1,662 @@
+# Automatic Speech Recognition using CTC
```

Comment:
Seems like this file should be reverted.

Reply:
Is this reverted now?  Not sure how this even got in there.